packages/toolset-revealer/output/index.html

```diff
@@ -0,0 +1,75 @@
+<html><head><meta http-equiv="refresh" content="1"></head><body><div id="valaa-inspire--main-container"><div id="valaa-inspire--main-root"><div><div><div class="inspire__lensMessage inspire__lensMessage_internalFailure"><p>There is an error with component:<button>Hide</button><button>Clear</button></p><pre style="font-family: monospace;">Invalid lens value when trying to render firstAbleDelegateLens, got value of type 'Timeout'
```

Comment:
Why was this file included? 
We should have the dev&test workflows such that generated, transpiled and other temporary input files are placed in dist/* somewhere, and any transient outputs ideally go under temp/* somewhere (although having outputs go under dist/* is also fine).
Otherwise the messiness of git status grows, the danger of accidentally including stuff in commits grows, the probability of depending on a file that exists on ones dev machine but doesn't exist on other machines grows, etc.

It's slightly tedious and a bit annoying to maintain toolchains so that source, inputs and outputs are so separated, but it's necessary to maintain sanity.
But knowing you can 'rm -rf dist' and everything still regenerates from scratch gives a unique type of peace of mind.